pkg/specs/validation_test.go

```diff
@@ -120,6 +120,7 @@ spec:
   user: "vagrant"
 `
 
+//nolint:unused
```

Comment:
Shall we open an issue to figure out how/if these should be used?

Reply:
I don't think it's important to track these in an issue right now, it's better to do that when we get to cleanup after the large structural changes have been merged.